oneview_redfish_toolkit/tests/test_util.py

```diff
@@ -265,20 +270,26 @@ def test_create_certs(self, oneview_client_mockup):
         self.assertTrue(os.path.exists(os.path.join("certs", "test" + ".key")))
 
     @mock.patch.object(util, 'OneViewClient')
+    @mock.patch.object(util, 'check_oneview_availability')
     @mock.patch('oneview_redfish_toolkit.util.delivery_retry_attempts', '')
     @mock.patch('oneview_redfish_toolkit.util.delivery_retry_interval', '')
-    def test_load_event_service_invalid_info(self, oneview_client_mockup):
+    def test_load_event_service_invalid_info(
+        self, check_ov_availability, oneview_client_mockup):
+        check_ov_availability.return_value = 0
+
         self.assertRaises(
             OneViewRedfishError, util.load_config(self.config_file))
 
+    @mock.patch.object(util, 'check_oneview_availability')
     @mock.patch.object(util, 'OneViewClient')
```

Comment:
Check if we still need this OneViewClient mock.

Reply:
We don't need it, once load_config() does not instantiate it anymore. Removing it.